src/twisted/mail/pop3.py

```diff
@@ -1493,7 +1514,9 @@ def sendLong(self, command, params):
         @param params: Command arguments.
         """
         if params:
-            self.sendLine('%s %s' % (command, params))
+            if not isinstance(params, bytes):
+                params = str(params).encode("utf-8")
```

Comment:
Same observation as above, a `test_BadCommand` that breaks `.encode("utf-8")` is interesting